validator/.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm --prefix validator install && npm --prefix validator run check
```

Comment:
can we don't do this one?

i'm very against not allowing to push because validations fail. There sometimes reasons to do that and this makes the experience a bit worst imho

Reply:
Oki, I'll remove it (I think that people won't run them otherwise, but the PR will fail, so it can be fixed here).